libraries/outputs.rb

```diff
@@ -24,6 +24,35 @@ def self.configure_outputs(node) # rubocop:disable Metrics/PerceivedComplexity,
         else
           [CernerSplunk.my_cluster(node)]
         end.each do |(cluster, bag)|
+          if bag['indexer_discovery'] == true
+            Chef::Log.warn "Configured ['receivers'] in cluster #{cluster} will be ignored since ['indexer_discovery'] is set to true." if bag['receivers']
+
+            indexer_discovery_settings = ((bag['indexer_discovery_settings'] && bag['indexer_discovery_settings']['outputs_configs']) || {}).reject do |k, _|
+              k.start_with?('_cerner_splunk')
+            end
+            output_stanzas["indexer_discovery:#{cluster}"] = indexer_discovery_settings
```

Comment:
I've been thinking about this, and I wonder if we need to run the various settings that we inject through the similar process we use for inserting procs to create values in conf files.

Reply:
For now, I think we can leave it like this. As mentioned in this comment (https://github.com/cerner/cerner_splunk/pull/158#discussion_r148015697) I believe the plan is to move forward with plain text pass4SymmKey.